### ISSUE TITLE
New-M365DSCDeltaReport - All resources are listed as different even though they are the same

### ISSUE THREAD

**Issue by micromega-prog** [microsoft/Microsoft365DSC#2256]
When I generate two configuration files using Export-M365DSCConfiguration for the same tenant, and then run:

New-M365DSCDeltaReport -Source C:\temp\Source\M365TenantConfig.ps1 -Destination C:\temp\Destination\M365TenantConfig.ps1 -OutputPath C:\Temp\TestReport.html -DriftOnly:$true

the report shows that all resources are different. The source and destination values for ResourceID are different for each resource
![image](https://user-images.githubusercontent.com/64819707/188739776-cb48e7a7-98f1-4e20-99d7-91e773c86ae0.png)

Is this the expected behaviour?

**Comment by andikrueger** [comment]
What version of M365DSC are you using? This issue was already raised in #1815, #1811 and should be closed with #1960

Please update to the latest version of M365DSC:

```powershell
Update-Module Microsoft365DSC
Update-M365DSCDependencies
Uninstall-M365DSCOutDatedDependencies
```

**Comment by micromega-prog** [comment]
Thank you, that fixed it. It looks like I may have had some outdated dependencies. Thanks again for your help!
